go/kbcmf/packets.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package kbcmf
+
+import ()
+
+type receiverKeysPlaintext struct {
+	GroupID    int    `codec:"gid"`
+	MACKey     []byte `codec:"mac,omitempty"`
+	SessionKey []byte `codec:"sess"`
+}
+
+type receiverKeysCiphertext struct {
+	KID  []byte `codec:"kid"`
```

Comment:
Changed to `key_id`, though you would be fine to old have the first 8 bytes of you wanted to.  The format is agnostic. But you need a corresponding `Keyring` object on your decryptor to get everything to work.